1. we don't need commons-daemon
2. jdt.jar is eclipse-ecj.jar which we provide as a separate package
3. javadoc would fail without network access
--- a/build.xml
+++ b/build.xml
@@ -1451,7 +1451,6 @@
   <target name="deploy" depends="package,build-docs,build-tomcat-jdbc,compile-webapp-examples"
           description="Default. Builds a working Tomcat instance">
 
-    <copy tofile="${tomcat.build}/bin/commons-daemon.jar" file="${commons-daemon.jar}" />
 
     <!-- Copy scripts -->
     <copy todir="${tomcat.build}/bin">
@@ -1512,7 +1511,6 @@
         <exclude name="${jdt.jar.filename}"/>
       </fileset>
     </delete>
-    <copy file="${jdt.jar}" todir="${tomcat.build}/lib"/>
 
     <!-- Add sources for examples -->
     <antcall target="examples-sources" />
@@ -2358,8 +2356,6 @@ Apache Tomcat ${version} native binaries for Win64 AMD64/EMT64 platform.
       <link href="../elapi"/>
       <link href="../websocketapi"/>
       <link href="../jaspicapi"/>
-      <link href="https://docs.oracle.com/javase/8/docs/api/"/>
-      <link href="https://javaee.github.io/javaee-spec/javadocs/"/>
       <packageset dir="${tomcat.dist}/src/java/">
         <include name="org/**"/>
         <exclude name="org/apache/el/parser/**"/>
